### PR TITLE
feat(collectorprofile): roughs in artists page

### PR DESCRIPTION
**Pull request by dzucconi**
Closes [DIA-535](https://artsyproduct.atlassian.net/browse/DIA-535)

[DIA-535]: https://artsyproduct.atlassian.net/browse/DIA-535?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/CollectorProfile/collectorProfileRoutes.tsx [hunk 1]
```diff
@@ -131,6 +141,13 @@ export const collectorProfileRoutes: AppRouteConfig[] = [
       }
     `,
     children: [
+      {
+        path: "artists",
```
Comment (dzucconi) [inline src/Apps/CollectorProfile/collectorProfileRoutes.tsx:145]: The route should be: `/collector-profile/artists`, not `/collector-profile/my-collection/artists`, to match the other routes.

File: src/Apps/CollectorProfile/Components/CollectorProfileArtists/CollectorProfileArtistsList.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      {loading ? (
+        new Array(10).fill(null).map((_, i) => {
+          return <CollectorProfileArtistsListArtistSkeleton key={i} />
+        })
+      ) : (
+        <>
+          {userInterestEdges.map(userInterestEdge => {
+            return (
+              <CollectorProfileArtistsListArtist
+                key={userInterestEdge.internalID}
```
Comment (araujobarret) [inline src/Apps/CollectorProfile/Components/CollectorProfileArtists/CollectorProfileArtistsList.tsx:58]: nit: I think we have be more specific, if this list component reuses the internalID we'll have duplicated keys being used

**Review by araujobarret — DISMISSED** [review]
I know sounds tedious to write unit test for a simple fragment like this, but that's the way we should work, with unit tests 😜

File: src/Apps/CollectorProfile/Components/CollectorProfileArtists/CollectorProfileArtistsList.tsx (hunk 2 above)
Comment (araujobarret) [reply src/Apps/CollectorProfile/Components/CollectorProfileArtists/CollectorProfileArtistsList.tsx:58]: nvm it's very uncommon for someone to use the `internalID` under any area of `CollectorProfileArtistsListArtist`. I was overthinking about the reuse of the same key (internalID).

**Pull request merged**